requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile requirements.in
+#    pip-compile
```

Comment:
The instructions on how to update these files were incorrect. Mind to review and merge https://github.com/mozilla/OpenWPM/pull/533 (assuming the tests pass) and then use those commands to re-compile the `.txt` files?

Reply:
Done